src/main/java/org/fogbowcloud/manager/core/intercomponent/xmpp/requesters/RemoteCreateOrderRequest.java

```diff
@@ -30,23 +30,30 @@ public Void send() throws Exception {
         return null;
     }
 
-    private IQ createIq() {
-        LOGGER.debug("Creating IQ for order: " + this.order.getId());
+    public IQ createIq() {
+        LOGGER.debug("Creating IQ for order: " + order.getId());
 
         IQ iq = new IQ(IQ.Type.set);
         iq.setTo(this.order.getProvidingMember());
         iq.setID(this.order.getId());
 
+        Element orderElement = marshallOrder(iq);
+        
+        LOGGER.debug("Jsonifying Order");
```

Comment:
In most of cases, logging without params does not help much. In think we can: i) remove this log line or ii) add the order to the log line (if it has a toString()) method.